src/Polly.Core/Simmy/Fault/FaultConstants.cs

```diff
@@ -2,5 +2,5 @@
 
 internal static class FaultConstants
 {
-    public const string OnFaultInjectedEvent = "OnFaultInjectedEvent";
+    public const string OnFaultInjectedEvent = "OnFaultInjected";
```

Comment:
How about? 

```suggestion
    public const string OnFaultInjectedEvent = "Chaos.OnFault"; 
```

Similar for others (`Chaos.OnLatency`, `Chaos.OnBehavior`).

The idea is to immediately know this is chaos event.

Reply:
Good idea, I will apply to all chaos strategies.

Reply:
Done